cluster/images/etcd/migrate-if-needed.sh

```diff
@@ -122,7 +122,6 @@ start_etcd() {
   ${ETCD_CMD} \
     --name="etcd-$(hostname)" \
     --debug \
-    --force-new-cluster \
```

Comment:
This looks good.  I noticed in the docs that this flag would be needed for a recover, but It doesn't look like we do any recovers in this script (double check me on that), so we should be okay. And we must remove it for HA cluster startup, as you've already discovered.